src/WebServerManager.cpp

```
@@ -272,6 +279,29 @@ void WebServerManager::handleUIDRegistrationPage() {
     _server.send_P(200, "text/html", UID_REGISTRATION_HTML);
 }
 
+void WebServerManager::handleLogViewer() {
+    _server.sendHeader("Access-Control-Allow-Origin", "*");
+    _server.send_P(200, "text/html", LOG_VIEWER_HTML);
+}
+
+void WebServerManager::handleApiLogs() {
```

Clang (14.0.6)

[warning] method 'handleApiLogs' can be made static

(readability-convert-member-functions-to-static)